failed to start project following Setup the Project in CONTRIBUTING.md

failed to start project following Setup the Project in CONTRIBUTING.md

## Describe the bug

```shell
± |develop ✓| → yarn dev
yarn run v1.22.10
$ NODE_ENV=development vite serve playground --config ./vite.config.ts
failed to load config from /home/ming/learn/js/chakra-ui/chakra-ui-vue-next/vite.config.ts
error when starting dev server:
Error: Cannot find module '/home/ming/learn/js/chakra-ui/chakra-ui-vue-next/node_modules/@chakra-ui/vue-auto-import/dist/cjs/index.js'
    at createEsmNotFoundErr (internal/modules/cjs/loader.js:912:15)
    at finalizeEsmResolution (internal/modules/cjs/loader.js:905:15)
    at resolveExports (internal/modules/cjs/loader.js:437:14)
    at Function.Module._findPath (internal/modules/cjs/loader.js:477:31)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:872:27)
    at Function.Module._load (internal/modules/cjs/loader.js:730:27)
    at Module.require (internal/modules/cjs/loader.js:957:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/home/ming/learn/js/chakra-ui/chakra-ui-vue-next/vite.config.ts:33:41)
    at Module._compile (internal/modules/cjs/loader.js:1068:30)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

```

## To reproduce

1. git clone repo
2. yarn bootstrap
3. yarn dev

## Additional context

i noticed that there are not `dist` folder in `chakra-ui-vue-next/node_modules/@chakra-ui/vue-auto-import`.

seems command `yarn bootstrap` does not aim to generate  above `dist` folder,  after i search lerna docment then run `lerna run build`,  `dist` is generated there and 'yarn dev' works

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -18,8 +18,16 @@ git clone https://github.com/<your_github_username>/chakra-ui-vue-next.git
 cd chakra-ui
 ```
 
-3. Setup all the dependencies and packages by running `yarn bootstrap`. This
+3. Build components
+```bash
+yarn build
+```
+
+4. Setup all the dependencies and packages by running `yarn bootstrap`. This
    command will install dependencies and bootstrap the repo using `lerna`
+```bash
+yarn bootstrap
+```
 
 > If you run into any issues during this step, kindly reach out to the Chakra UI
 > Vue team here:[https://discord.gg/cMpMfvxa](https://discord.gg/cMpMfvxa)

